### DISCUSSION TITLE
failed sending /srv/tftp/ltsp/undionly.kpxe to XXX.XXX.XXX.XXX

### DISCUSSION THREAD

**Discussion by CarlosAPCunha** [ltsp/ltsp#276] [Q&A] [answered]
Hi!

When I start the station it takes IP by DHCP, but an error occurs, it follows images and a video.

![Captura de Tela_Área de Seleção_20200518122259](https://user-images.githubusercontent.com/65431346/82230610-5566b480-9902-11ea-9ce7-48fce2d2c8c8.png)

In video(zip file) after erro the SO  loading is "thin client hardware", not ltsp...
[IPXE_ERROR.mp4.zip](https://github.com/ltsp/community/files/4645057/IPXE_ERROR.mp4.zip)

Syslog error:

> 

error 0 TFTP Aborted received from XXXX
failed sending /srv/tftp/ltsp/undionly.kpxe to XXX

>

Information more:

>
ltsp info
LTSP version: 20.04-1 ~ ubuntu18.04.1

SERVER XSESSIONS: ls: cannot access '/ usr / share / xsessions': No such file or directory
LTSP command failed: ls / usr / share / xsessions

SERVER OS: PRETTY_NAME = "Ubuntu 18.04.4 LTS"

CHROOTS:
bionic

VMs:

IMAGES:

>

regrads;

**Comment by alkisg** [comment, marked as answer]
Hi, try these commands on the server:

```shell
sudo -i
cd /srv/tftp/ltsp/
mv undionly.kpxe undionly.kpxe-bad
wget boot.ipxe.org/ipxe.pxe -O undionly.kpxe
```

Does the client boot after that?

**Comment by CarlosAPCunha** [comment]
Hi!
now it's working
thanks again.

:-D

regards;

**Comment by CarlosAPCunha** [comment]
Only information in the logs still see message:

failed sending /srv/tftp/ltsp/undionly.kpxe to xxx
sent /srv/tftp/ltsp/undionly.kpxe to xxx

but no more problems...

Regards;

**Comment by alkisg** [comment]
Yeah that's a usual message for dnsmasq, it's not a problem.
The problem was that undionly.kpxe didn't work correctly for your network adapter.
If you have time, you could file a bug in ipxe.org.

**Comment by CarlosAPCunha** [comment]
Ok, thanks so much.

Regards;
